### DISCUSSION TITLE
mvn verify many errors

### DISCUSSION THREAD

**Discussion by Laccio** [apache/seatunnel#5728] [Q&A]
Hello everyone, I believe I haven't fully grasped the functioning pipeline of Seatunnel. When I run `./mvnw verify`, I encounter a series of errors. I'm using Seatunnel version 2.3.3.

I've also downloaded TestContainers, which I understand is necessary for running end-to-end tests. What am I missing to make them work? One of a kind errors i get is the following:
![image](https://github.com/apache/seatunnel/assets/44461417/b22ea908-0a17-4048-8705-eda69804fcd4)

**Comment by liunaijie** [comment]
you need to import the redis connector first
